core/server/web/api/v2/admin/routes.js

```diff
@@ -195,8 +199,6 @@ module.exports = function apiRoutes() {
         apiv2.http(apiv2.upload.image)
     );
 
-    router.post('/db/backup', mw.authenticateClient('Ghost Backup'), api.http(api.db.backupContent));
```

Comment:
Moved this route up to the rest of `/db` endpoints so they are grouped nicely

Reply:
👍